components/brave_ads/browser/BUILD.gn

```diff
@@ -32,6 +32,8 @@ source_set("browser") {
     "//components/domain_reliability",
     "//content/public/browser",
     "//content/public/common",
+    # for session_tab_helper.h
```

Comment:
How about putting some comments something like `Remove these chrome layer dependencies`? because we want to decouple component and chrome.

Reply:
+1